src/pages/ClassComponent.mdx

```diff
@@ -117,7 +117,8 @@ export class ChildClassComponentWithObjectProps extends Component<
 ```
 
 When parent component changes `obj.num` and doesn't change `obj.str`, the component
-still re-renders.
+still re-renders. This will only be true if the actual instance changed,
```

Comment:
I am a little confused by this, this section talks about ClassComponent, not PureComponent.

Reply:
you are right, this shouldn't be here, removed.

Reply:
I verified, there is no broken links in TMP